@DQ/Q8.m

```diff
@@ -20,7 +25,13 @@
 % DQ Robotics website: dqrobotics.github.io
 %
 % Contributors to this file:
-%     Bruno Vihena Adorno - adorno@ufmg.br
+%
+% 1. Bruno Vilhena Adorno (adorno@ieee.org)
+%        - Responsible for the original implementation.
+%
+% 2. Juan Jose Quiroz Omana (juanjqo@g.ecc.u-tokyo.ac.jp)
+%        - Removed Q4() from Q8.m and exposed the method to the user.
```

Comment:
@juanjqo In this case I'd just say "removed Q4()" because this update is related to this specific file. If you could add the link to the issue/pr it would also be nice.

Both comments are simply aiming for us to further (and continuously) refine the documentation of changes.

Reply:
@mmmarinho Thanks for the comments. I updated the documentation according to your suggestions.